src/wizards/deployToOpenShift/deployToOpenShift.ts

```diff
@@ -0,0 +1,51 @@
+import { commands, window } from "vscode";
+import { ProjectLabelInfo } from "../../definitions/ProjectLabelInfo";
+import { getOpenShiftConnector, installOpenShiftConnector, isOpenShiftConnectorInstalled } from "../../utils/openShiftConnectorUtils";
+import { getQuarkusProject } from "../getQuarkusProject";
+
+/**
+ * Attempts to select a Quarkus project, then deploy it to OpenShift using OpenShift Connector
+ */
+export async function deployToOpenShift() {
+  try {
+    await installOpenShiftConnectorIfNeeded();
+    const quarkusProject: ProjectLabelInfo = await getQuarkusProject();
+    await commands.executeCommand("openshift.component.createFromRootWorkspaceFolder", quarkusProject.uri, "java-quarkus");
```

Comment:
Once a cluster is already available (or else it [fails](https://github.com/redhat-developer/vscode-openshift-tools/issues/1849)), the command let's you create an [application](https://github.com/redhat-developer/vscode-openshift-tools/issues/1850) and a component, if needed.

Then it fails with:

<img width="539" alt="Screenshot 2020-11-13 at 18 03 23" src="https://user-images.githubusercontent.com/148698/99099398-ac5ff480-25da-11eb-978b-5e85b4513a03.png">

Either the command call is missing something, or openshift-tools has an issue. @dgolovin which is it?

Reply:
@fbricon will look at this, it could be odo failure, bz project already has devfile.yaml. I am going to fix it by detecting project with devfile.yaml in root folder and add option `--devfile devfile.yaml`, so it will just use existing file.

Reply:
@fbricon https://github.com/redhat-developer/vscode-openshift-tools/issues/1815 is fixed. Released to master branch.

Reply:
@dgolovin Thanks for the update! I looked over the code, and it seems I was just calling the command with the wrong parameters. It should work now. I'll record a short demo of the new workflow.